app/client/websocket/index.ts

```diff
@@ -87,11 +99,30 @@ class WebSocketClient {
                 return;
             }
 
-            this.conn = new WebSocket(connectionUrl, [], {headers: {origin}, ...(additionalOptions || {})});
+            let url = connectionUrl;
+            const config = getConfig(Store.redux?.getState());
+            const reliableWebSockets = config?.EnableReliableWebSockets === 'true';
+
+            if (reliableWebSockets) {
```

Comment:
There's no need to really add this condition. Older servers will simply ignore the query params. Keeps the code simple. 0/5.

Reply:
I'd prefer to keep it if that's ok